web-console/future-openshift/custom-catalogs/custom-catalogs.md

```diff
@@ -0,0 +1,66 @@
+# Adding custom catalogs to OperatorHub
+
+Admins can create custom catlogs in order to give admin console users the ability to add operators from other sources, beyond the default OperatorHub operator sources.
+
+## Global Configuration
+
+OperatorHub sources are found through: Administation > cluster settings > global configuration > OperatorHub / sources tab.
+
+![global config](img/operatorhub-custom-catalog1.png)
+
+![OperatorHub config](img/operatorhub-custom-catalog2.png)
+
+## Sources
+
+In the OperatorHub configuration, users can see the current default sources that are feeding into OperatorHub. These sources are technically "Operator Sources" and have a resource badge for "OS". Users can also create a catalog source from here, which would create a "catalog source" resource. From a user perspective, both of these sources are accomplishing the goal of populating OperatorHub with operators that they can leverage and keep updated. They will only be able to create catalog sources (not operator sources) from the UI, as the goal is to deprecate the use of operator sources in favor of using catalog sources.
+
+![OperatorHub sources default](img/operatorhub-custom-catalog3.png)
```

Comment:
Like discussed earlier, I propose to visually separate the built-on catalog sources from the custom ones. Giving them their own section. This is due to their different behavior (can't be deleted, only disabled) and different backend implementation (`OperatorSource` vs. `CatalogSource`).

Reply:
In a conversation to better understand the technical aspects with @kevinrizza and @TheRealJon it seems like we can just show the catalog source, because both a catalog and operator source exists for the default catalogs – this will help keep things more straightforward if we can just show one type in the list.

Reply:
That's true because an `OperatorSoure` in turn creates a `CatalogSource` (long story). The API you are dealing with to disable the default catalogs however deals at the `OperatorSource` level. Would the UI be so clever to cope with that?

Reply:
I think it makes sense to allow users to disable these from the catalog source (even if behind it, it disables the operator source). We could have a short description about this in the confirmation dialog if you think it would be confusing to simplify this from the perspective of users who are very familiar with this operatorsource vs. catalogsource relationship (this could give enough transparency to the technical aspect but still provide a relatively simple experience)